zerver/lib/bugdown/fenced_code.py

```diff
@@ -282,6 +314,19 @@ def format_quote(self, text):
             quoted_paragraphs.append("\n".join("> " + line for line in lines if line != ''))
         return "\n\n".join(quoted_paragraphs)
 
+    def format_tex(self, text):
+        # type: (Text) -> Text
+        paragraphs = text.split("\n\n")
+        tex_paragraphs = []
+        for paragraph in paragraphs:
+            html = render_tex(paragraph, is_inline=False)
+            if html is not None:
+                tex_paragraphs.append(html)
+            else:
+                tex_paragraphs.append('<span class="tex-error">' +
+                                      cgi.escape(paragraph) + '</span>')
```

Comment:
Maybe use `from django.utils.html import escape`?  We already use that module, and it'll handle the Python 3 transition nicely.